build.gradle

```diff
@@ -27,4 +27,9 @@ if (!ossUser.isEmpty() && !ossPass.isEmpty()) {
             }
         }
     }
+}
+
+// buildSrc tests fail occasionally without running clean first
+build {
+    dependsOn clean
```

Comment:
It is technically new code that resolved an issue seen during the merge.  It can also occur on 6.2.x, so I will commit it over there and then update this PR before merging.